cv32/sim/uvmt_cv32/xrun.mk

```diff
@@ -252,13 +252,6 @@ custom: $(XRUN_SIM_PREREQ) $(CUSTOM_DIR)/$(CUSTOM_PROG).hex
 
 ################################################################################
 # Explicit target tests
-debug_test: $(XRUN_SIM_PREREQ) $(CORE_TEST_DIR)/debug_test/debug_test.hex
```

Comment:
I do not understand why you would remove this target.

Reply:
Since I changed the test to use the new yaml setup I figured it would be less to maintain if I removed the explicit target. I think the interrupt test and some of the random tests also disappeared when converted to yaml?

Reply:
OK, I hadn't realized you'd converted to the yaml flow.